### PR TITLE
Make mod files compatible with C/C++ code generation in NEURON

### PR DESCRIPTION
**Pull request by iomaganaris**

### CODE REVIEW

File: sim/Gfluct.mod [hunk 1]
```diff
@@ -179,11 +179,6 @@ PROCEDURE oup() {		: use Scop function normrand(mean, std_dev)
 
 VERBATIM
 #include "nrnran123.h"
```
Comment (olupton) [inline sim/Gfluct.mod:181]: We can drop this too, https://github.com/neuronsimulator/nrn/pull/1755 means that it is implicitly included.

File: sim/orn.mod [hunk 2]
```diff
@@ -201,15 +201,8 @@ static int _ran_compat; /* specifies the noise style for all instances */
 #endif /* running in NEURON */
 ENDVERBATIM
 
-
 VERBATIM
 #include "nrnran123.h"
```
Comment (olupton) [inline sim/orn.mod:205]: As above, can be dropped.

File: sim/Gfluct.mod (hunk 1 above)
Comment (olupton) [reply sim/Gfluct.mod]: Done in 399f538654fd91f9152906e95e51b3de5702c28c.

File: sim/orn.mod (hunk 2 above)
Comment (olupton) [reply sim/orn.mod]: Done in 399f538654fd91f9152906e95e51b3de5702c28c.

**Review by olupton — APPROVED** [review]

**Review by pramodk — APPROVED** [review]

**Pull request merged**